Apps/W1/HybridGP/app/src/Migration/Support/GPConfiguration.table.al

```diff
@@ -104,14 +104,16 @@ table 4024 "GP Configuration"
     end;
 
     procedure IsAllPostMigrationDataCreated(): Boolean
+    var
+        GPCompanyAdditionalSettings: Record "GP Company Additional Settings";
     begin
         exit(
-                "CheckBooks Created" and
-                "Open Purchase Orders Created" and
                 "Fiscal Periods Created" and
-                "Vendor EFT Bank Acc. Created" and
-                "Vendor Classes Created" and
-                "Customer Classes Created"
+                (not GPCompanyAdditionalSettings.GetBankModuleEnabled() or "CheckBooks Created") and
```

Comment:
glad you wrote this, my BC and\or statement writing could use some work...I would have just used a number of exit(value) statements. ;)